articles/building-a-basic-api-with-nodejs/index.md

```diff
@@ -0,0 +1,220 @@
+
+Finally Opening `http://localhost:3000/list_movies` should return the same data found in the `movies.json` file, as expected.
+
+Stop the server by pressing `CTRL + C`.
+
+Congratulations!!! You have managed to build and locally serve a Node.js REST API built with Express!
+
+In the next step, you will expose the server to the internet with Ngrok.
+
+### Step 3 — Expose Server with Ngrok
+In this step, you are going to use Ngrok to expose your localhost server to the internet.
+
+First, get the Express server up and running.
+
+Run `node index.js`
+
+Next, you'll need to expose the port the server is running on.
+
+Which in our case is `port 3000`.
+
+To do this we need to run the `ngrok` executable.
+
+In the directory where the `ngrok` executable is, run the command below:
+
+```bash
+./ngrok http 3000
+```
+
+If successful you should see something similar to the snippet below.
+
+```bash
+ngrok by @inconshreveable                                                                                                    (Ctrl+C to quit)                                                                                                                                             Session Status                online
+Session Expires               7 hours, 59 minutes
+Version                       2.3.35
+Region                        United States (us)
+Web Interface                 http://127.0.0.1:4040
+Forwarding                    http://367f23f4ca81.ngrok.io -> http://localhost:3000
+Forwarding                    https://367f23f4ca81.ngrok.io -> http://localhost:3000
+
+Connections                   ttl     opn     rt1     rt5     p50     p90
+                              0       0       0.00    0.00    0.00    0.00
+```
+
+The snippet above might be a little overwhelming but once we break it down you'll feel significantly more comfortable with it.
+
+The first thing you need to note is the URLs to the right of the word "Forwarding".
+
+Both are HTTP endpoints.
+
+The only difference is one (HTTPS) uses the [SSL protocol](https://www.ibm.com/support/knowledgecenter/en/SSYKE2_7.1.0/com.ibm.java.security.component.71.doc/security-component/jsse2Docs/ssloverview.html) to ensure security.
+
+The next thing to note is the URL to the right of "Web Interface" which is a web variant of what you see above.
+
+The next thing is the counter to the right of "Session Expires" which is how long the exposed URLs will be valid for if you don't stop the Ngrok service.
+
+Finally, once you start making requests to the server a listing of every request will appear under the heading "HTTP Requests".
+
+You are now ready to test out your server!!!
+
+In the next step, you will use postman to test out your server.
+
+### Step 4 — Test Requests with Postman
+In this step, you are going to use postman to make test requests to your running server.
+
+On postman create a new request tab and make a GET call to your /list_movies endpoint.
```

Comment:
While you do have an image that shows the example URL in Postman, it would be nice if you explained things more here especially for readers using screen readers who won't be able to see the screenshot. 

Something like "the endpoint is your ngrok URL followed by /list_movies."

Reply:
noted